# Searching

To search just start typing a variant in the search box and it will auto complete. Variants are identified using [HGVS nomenclature](http://www.hgvs.org/mutnomen/), a standard created by the Human Genome Variation Society.  Most genetic test results are reported in HGVS nomenclature. This nomenclature describes a variant by indicating (1) a reference sequence, (2) what kind of sequence it is (genomic, cDNA or protein), (3) the position of the variant in relation this reference sequence, and (4) the nucloetide or protein differences between the variant and the reference sequence.  For example, _NM_007294.3:c.5053A>G_ indicates a variant relative to reference sequence _NM_007294.3_, which is a cDNA sequence (as indicated by _c_), that the variant is in position 5053, and that it involves changing an _A_ to a _G_.

# Table Columns

#### Mutation category (BIC)

Transcript context of the mutation.

#### PolyPhen score

PolyPhen score.  A score closer to 1 suggests a damaging variant

#### SIFT score

SIFT score, indicating if the variant is tolerated, deleterious or other

#### African Allele Frequency (1000 Genomes)

African-American minor allele frequency, from 1000 Genomes

#### Allele Frequency

Allele frequency, estimated from the combination of ExAC, ESP and 1000 Genomes allele frequences

#### Allele Frequency (1000 Genomes)

Overall allele frequency, from 1000 Genomes

#### Allele Frequency (ExAC)

Minor allele frequency

#### AMR Allele Frequency (1000 Genomes)

Allele frequency in the Admixed American population, from 1000 Genomes

#### EAS Allele Frequency (1000 Genomes)

Allele frequency in the East Asian population, from 1000 Genomes

#### EUR Allele Frequency (1000 Genomes)

European allele frequency in the European population, from 1000 Genomes

#### Maximum Allele Frequency
Highest allele frequency reported by any data source, for any population.

#### Allele Frequencies: EA|AA|All (ESP)
Allele frequencies from ESP, expressed as EA (European)|AA (African American)|All

#### SAS Allele Frequency (1000 Genomes)
Allele frequency in the Southeast Asian population, from 1000 Genomes

#### Variant Frequency (LOVD)
Allele frequency, from LOVD, expressed as a fraction relative to the reference population

#### BIC Variant Identifier
Variant identifier in BIC nomenclature

#### Protein
HGVS protein identifier

#### SCV Accession (ClinVar)
SCV accession in the ClinVar repository

#### Nucleotide
HGVS string that represents the variant at cDNA nucleotide level

#### Other HGVS Nucleotide(s)
Other HGVS cDNA strings observed for this variant

#### Protein Amino Acid Change
Amino acid change, in single-letter amino acid format and protein position

#### Reference cDNA Sequence
Sequence on which the variant was observed

#### Gene Symbol
Gene that the variant maps to (BRCA1 or BRCA2).

#### Genome (GRCh38)
Coordinate of the variant on the GRCh38 reference genome

#### Genome (GRCh36)
Coordinate of the variant on the GRCh36 reference genome

#### Genome (GRCh37)
Coordinate of the variant on the GRCh37 reference genome

#### Allele Origin (ClinVar)
Allele origin, indicating if the variant is germline or somatic in origin.  From ClinVar.

####  Allele Origin (ENIGMA)
Allele origin, indicating if the variant is germline or somatic in origin.  From ENIGMA.

####  Ethnicity (BIC)
Ethnicities of the observed patients, from BIC

#### Allele Origin (BIC)
Origin of the allele, Germline (G) or Somatic (S).  From BIC

#### Allele Origin (LOVD)
Allele origin, from LOVD

#### Patient Nationality (BIC)
Nationality of the patient associated with the variant.  From BIC

#### Variant Haplotype (BIC)
Haplotype (example: FA FANCD1_00037)

#### Family members carrying this variant (BIC)
Number of family members carrying this variant.  From BIC.

#### Co-occurrence likelihood (exLOVD)
The likelihood ratio based on the frequency of co-occurrence between the variant of interest and clearly
 pathogenic variants in the same gene. From exLOVD.

#### Prior probability of pathogenicity (exLOVD)
The combined prior probability in favor of pathogenicity. is a combination of the missense analysis prior
 probability and the splicing analysis prior probability. Generally, it is the higher of these two prior probabilities.  From exLOVD.

#### Missense analysis probability of pathogenicity (exLOVD)
This prior probability estimate combines position in the protein with an evaluation of missense substitutions
 that fall in the proteins key functional domains.  From exLOVD.

#### Probability of pathogenicity (exLOVD)
Posterior probability of pathogenicity. From exLOVD.

#### Segregation Likelihood Ratio (exLOVD)
The likelihood ratio based on segregation analysis

#### Summary Family History Likelihood Ratio (exLOVD)
The likelihood ratio based on an analysis of the severity of summary family histories of breast and/ or ovarian
 cancer.

#### Assertion Method (ENIGMA)
URL of the ENIGMA classification rules

#### Clinical Significance Citation (ENIGMA)
Pubmed ID on this clinical significance

#### Literature Reference (BIC)
Literature Reference(s).  From BIC

#### Literature Reference (exLOVD)
Literature Reference(s).  From exLOVD.

#### Assertion Method (ENIGMA)
Basis of the evaluation (ENIGMA variants , invariant)

#### Clinical Significance (BIC)
Clinical significance, from BIC.  This ranges from Class 1 to Class 5, where Class 1 is Benign and Class 5 is
 Pathogenic.

#### Clinical Importance (BIC)
Clinical significance (yes|no|pending|unknown)

#### Clinical Significance (ClinVar)
Clinical significance according to ClinVar

#### Clinical Significance (ENIGMA)
Clinical significance according to ENIGMA

#### Collection Method (ENIGMA)
Method by which ENIGMA arrives at its clinical classificaitons

#### Comment on Clinical Significance (ENIGMA)
Comment on how the derivation of the IARC class, from ENIGMA

#### Date last evaluated (ENIGMA)
Date at which the variant was evaluated by ENIGMA

#### Date last updated (ClinVar)
Date the variant was last updated in ClinVar

#### Has Discordant Evidence
Indicates if there is evidence of a discordant classification for this variant

#### Functional Analysis Result (LOVD)
Functional analysis or prediction of the impact of this variant.  From LOVD

#### Functional Analysis Method (LOVD)
The method used in analyzing variant function.  From LOVD.

#### Analysis Method (ClinVar)
Method by which the significance of the variant was determined.  From ClinVar

#### Pathogenicity
Pathogenicity, as reported by ENIGMA, ClinVar and BIC.

#### ClinVar Accession
ClinVar SCV accession

#### Condition Category (ENIGMA)
Condition Type (ENIGMA)

#### Condition ID Type (ENIGMA)
Indicates if this variant came from OMIM.  From ENIGMA

#### Condition ID Value (ENIGMA)
Indicates the disease association for this variant.  From ENIGMA.

#### Source(s)
List of repositories containing the variant

#### Source URL(s)
URL(s) pointing back to the original source data

#### Submitter (ClinVar)
Submitting organization.  From ClinVar.

#### URL (ENIGMA)
URL listing the variant.  From ENIGMA.


 
# Variant Detail Page Glossary

#### Gene symbol
Gene name, as named by [HGNC](http://www.genenames.org/).

#### Genomic Coordinate
Coordinate of the variant on the genome.

#### Reference sequence
The reference sequence corresponding to the HGVS coordinates indicated in the HGVS_cDNA column, e.g. NM_000492.3, NG_016465.3, or NC_000007.13. 

#### HGVS_cDNA
[HGVS coordinates](#HGVS) on the cDNA.

#### BIC Nomenclature
BIC name for this variant.

#### Abbrev AA Change
The amino acid change introduced by the variant, using abbreviated (one-letter) amino acid names.

#### URL
A link with further information on this variant.

#### Condition ID type
The Condition ID type, together with the Condition ID value, indicate the condition associated with this variant according to a standard ontology. Condition ID types can be: OMIM, MeSH, MedGen, UMLS, Orphanet, HPO.

#### Condition ID value
The Condition ID value, together with the Condition ID type, indicate the condition associated with this variant according to a standard ontology.  The condition ID type specifies an ontology, while the condition ID value indicates the term within that ontology that identifes this condition.

#### Condition category
Human-readable condition, describing the biological impact of the variation.

#### Clinical significance
Possible values are Pathogenic, Likely pathogenic, Uncertain significance, Likely benign, Benign, association, drug response, confers sensitivity, protective, risk factor, other, not provided.

#### Date last evaluated
The date on which the clinical significance of the variant was last evaluated by the submitter.

#### Assertion method
Describes the method and criteria that were used to make the assertion of clinical significance. 

#### Assertion method citation
Citation or URL describing the method and criteria used to make assertions of clinical significance.  

#### Clinical significance citations
Citations documenting the clinical significance. Can be from PubMed, PubMedCentral, DOI, or NCBI Bookshelf.

#### Citations or URLs for clinical significance without database identifiers
Citations that require a URL, or that do not have an identifier in one of the resources indicated in the Clinical significance citations column.

#### Comment on clinical significance
Describes the rationale for the clinical significance.

#### Collection method
Method used to collect the data that supports the assertion of clinical significance. Allowed values: case-control, clinical testing, literature only, reference population, research.

#### Allele origin
Variants are classified as either _germline_ or _somatic_, depending on how they are acquired.  _Germline_ variants are genetic changes that we inherit from our parents.  _Somatic_ variants are DNA changes that we acquire over our lifetime, often through exposure to pollutants, toxins, radiation and other carcinogens.  

#### Affected status
Indicates whether or not the individual(s) had the condition.

#### ClinVarAccession
Accession number in the ClinVar database.

#### Novel or Update
Indicates whether the submission to ClinVar is novel (and accessions were reserved prior to submission), an update to an existing record.

#### HGVS_protein
[HGVS coordinates](#HGVS) on the protein.


# Lollipop Plots

Lollipop plots are a tool to visualize the chromosomal position and pathogenicity classification for each variant in a gene.  Here, each circle-capped 'lollipop' indicates whether a BRCA1 and BRCA2 variant is benign (labeled in blue) or pathogenic (labeled in red).  The diameter of the circle on top of each lollipop indicates the number of unique variants that have been observed at an individual locus.  The bottom bar that runs along the x-axis of the diagram displays the position of each exon in the selected gene.  To alternate between the BRCA1 and BRCA2 lollipop charts, click the *Select Gene* button located in the upper-left hand corner of the page.
